tests/cases/compiler/expandoFunctionContextualTypesNoValue.ts

```diff
@@ -0,0 +1,6 @@
+// GH #38532
+import Foo from "blah";
+
+export function Foo() { }
+
+Foo.bar = () => { };

```

Comment:
Does this need to actually have a parameter to invoke the contextual typing machinery?

Reply:
The bug report had one, but it got there with or without an argument.  (And adding a second test file seems like an overkill for a tiny fix...)